bokehjs/test/integration/glyphs/webgl.ts

```diff
@@ -58,4 +58,27 @@ describe("webgl", () => {
 
     await display(row([p0, p1]))
   })
+
+  it("should not render size 0 markers", async () => {
```

Comment:
There needs to be issue number associated with regression tests (as a part of the description). Preferably this should be moved under regression tests.

Reply:
So, that I can run `node make test:integration -k '#12357'`.

Reply:
Done.